app/code/Magento/GroupedProduct/Api/Data/GroupedItemQtyInterface.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\GroupedProduct\Api\Data;
+
+/**
+ * Object that contains quantity information for a single associated product of a Grouped Product
+ *
+ * Interface \Magento\GroupedProduct\Api\Data\GroupedItemQtyInterface
+ */
+interface GroupedItemQtyInterface extends \Magento\Framework\Api\ExtensibleDataInterface
```

Comment:
Please, remove all setters from the interface except `setExteensionAttributes`. You may see an example [here](https://github.com/magento/inventory/blob/c90749314415076d3df316d55de165210418f009/InventoryInStorePickupSalesApi/Api/Data/ResultInterface.php#L17).